metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/WorkflowExecutor.java

```diff
@@ -364,37 +359,4 @@ private void preparePluginStateAndFinishedDate(AbstractMetisPlugin abstractMetis
     }
     workflowExecutionDao.updateWorkflowPlugins(workflowExecution);
   }
-
-  private void periodicCheckingLoopMocked(long sleepTime, AbstractMetisPlugin abstractMetisPlugin) {
-    for (int i = 1; i <= MONITOR_ITERATIONS_TO_FAKE; i++) {
-      try {
-        if (workflowExecutionDao.isCancelling(workflowExecution.getId())) {
-          // Update workflowExecution first, to retrieve cancelling information from db
-          workflowExecution = workflowExecutionDao.getById(workflowExecution.getId().toString());
-          return;
-        }
-        Thread.sleep(sleepTime);
-        fakeMonitorUpdateProcessedRecords(abstractMetisPlugin, i);
-        Date updatedDate = new Date();
-        abstractMetisPlugin.setUpdatedDate(updatedDate);
-        workflowExecution.setUpdatedDate(updatedDate);
-        workflowExecutionDao.updateMonitorInformation(workflowExecution);
-      } catch (InterruptedException e) {
-        LOGGER.warn("Thread was interrupted", e);
-        Thread.currentThread().interrupt();
-        return;
-      }
-    }
-    abstractMetisPlugin.setFinishedDate(new Date());
-    abstractMetisPlugin.setPluginStatusAndResetFailMessage(PluginStatus.FINISHED);
-    workflowExecutionDao.updateWorkflowPlugins(workflowExecution);
-  }
-
-  private void fakeMonitorUpdateProcessedRecords(AbstractMetisPlugin abstractMetisPlugin,
-      int iteration) {
-    ExecutionProgress executionProgress = abstractMetisPlugin.getExecutionProgress();
-    executionProgress.setExpectedRecords(MONITOR_ITERATIONS_TO_FAKE * FAKE_RECORDS_PER_ITERATION);
```

Comment:
You have not removed the two constants: they are no longer needed.

Reply:
Fixed